### PR TITLE
feat(JOML): migrate block placement

### PR DESCRIPTION
**Pull request by pollend**

### CODE REVIEW

**Comment by jdrueckert** [comment]
Requires changes in
- [ ] Minimap
- [ ] StructuralResources
- [ ] Furnishings
- [ ] Rails
- [ ] MultiBlock
- [ ] IRLCorp
- [ ] Smithing
- [ ] MobileBlocks
- [ ] ModularComputers
- [ ] FlowingLiquids
- [ ] Fluid
- [ ] StructureTemplates
- [ ] ItemPipes
- [ ] Climbables
- [ ] ChangingBlocks
- [ ] DynamicCities
- [ ] GrowingFlora
- [ ] AnotherWorldPlants

File: engine/src/main/java/org/terasology/world/block/entity/placement/BlockPlacingSystem.java [hunk 1]
```diff
@@ -34,6 +38,9 @@
 
     @ReceiveEvent(components = {WorldComponent.class}, priority = EventPriority.PRIORITY_TRIVIAL)
     public void placeBlockInWorld(PlaceBlocks event, EntityRef world) {
-        worldProvider.setBlocks(event.getBlocks());
+        //   worldProvider.setBlocks(event.getBlocks());
```
Comment (jdrueckert) [inline engine/src/main/java/org/terasology/world/block/entity/placement/BlockPlacingSystem.java:41]: do we still need this? if so why is it commented out? if not, why is it still there :yum:

**Review by jdrueckert — CHANGES_REQUESTED** [review]

File: engine/src/main/java/org/terasology/world/block/entity/placement/BlockPlacingSystem.java (hunk 1 above)
Comment (pollend) [reply engine/src/main/java/org/terasology/world/block/entity/placement/BlockPlacingSystem.java:41]: looks like the same line but commented out :P

Comment (pollend) [reply engine/src/main/java/org/terasology/world/block/entity/placement/BlockPlacingSystem.java:41]: opps

**Review by jdrueckert — APPROVED** [review]

**Pull request merged**